.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,html,css}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```

Comment:
This file tells your IDE how it should ident the files.
It's really useful when working with multiple developers.

You can check if your IDE already has it or needs a plugin here:
https://editorconfig.org/